nicegui/native/native_mode.py

```diff
@@ -47,7 +65,12 @@ def _open_window(
     closed = Event()
     window.events.closed += closed.set
     _start_window_method_executor(window, method_queue, response_queue, closed)
-    webview.start(storage_path=tempfile.mkdtemp(), **core.app.native.start_args)
+    webview.start(
+        func=bind,
+        args=(window, drop_queue),
+        storage_path=tempfile.mkdtemp(),
+        **core.app.native.start_args,
+    )
```

Comment:
```suggestion
    webview.start(bind, (window, drop_queue), storage_path=tempfile.mkdtemp(), **core.app.native.start_args)
```

Let's keep it short and concise.

Reply:
I don't really know so I asked Claude and got this answer:
````
The window.dom.document.events.drop is an event from pywebview that expects to have handlers added to it, but mypy sees it as a standard Python Event (like from the threading module) which has a different expectation for its __iadd__ method.
```

I added `# type: ignore[arg-type]` after the string but it becomes too long for autopep8 so it places the comment on a new line.
I saved the file without formatting but for it to save with auto formatting I think we'll have to split the line up into multiple lines to let `# type: ignore[arg-type]` be on the correct line.
Should I do that?